pkg/state/context.go

```diff
@@ -49,10 +58,31 @@ type State struct {
 	UpgradeMachineDeployments bool
 	PatchCNI                  bool
 	CredentialsFilePath       string
+
+	lock *sync.Mutex
+}
+
+func (s *State) KubeAdmVerboseFlag() string {
+	if s.Verbose {
+		return "--v=6"
+	}
+	return ""
 }
 
 // Clone returns a shallow copy of the State.
 func (s *State) Clone() *State {
 	newState := *s
+	newState.lock = &sync.Mutex{}
 	return &newState
 }
+
+func GenerateBootstrapToken(s *State) error {
+	s.lock.Lock()
```

Comment:
What is the reason for the lock here? Is this concurrect somewhere?

Reply:
Not currently :) But I now have better idea, so I'd remove this.